### PR TITLE
chore: ensure test services are up-to-data

### PR DESCRIPTION
**Pull request by jjtang1985**
## Context

The pipeline now generates the test services (v2 and v4 but without the ones used in e2e tests) to check whether they are up-to-date.

## Definition of Done

Please consider all items and remove only if not applicable.

- [ ] Tests created/adjusted for your changes.
- [ ] Release notes updated.
  * Provide sufficient context so that each entry can be understood on its own.
  * Be specific about names of functions, classes, modules, etc.
  * Describe when or where this is relevant
  * Use indicative and present tense. For example, write "Provide function `name` that does X in order to Y" over "Now X can be done by calling a new function".
- [x] PR title adheres to [conventional commit guidelines](https://www.conventionalcommits.org) (please note that only `fix:` and `feat:` will end up in the release notes)
- [ ] If applicable: Properly documented (JSDoc of public API)
- [ ] If applicable: Check if `node run doc` still works.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,6 +48,7 @@
     "check:dependencies": "yarn workspaces run check:dependencies",
     "check:license": "node scripts/check-licenses.js",
     "check:vulnerability": "yarn audit-ci -h",
+    "check:test-service": "yarn generate:test-services:v2 && yarn generate:test-services:v4 && yarn ts-node scripts/check-test-service.ts",
```
Comment (FrankEssenberger) [inline package.json:51]: Minor: Just a matter of taste. You could add the generation of the test services also to the typescript file.

File: scripts/check-test-service.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
```
Comment (FrankEssenberger) [inline scripts/check-test-service.ts:1]: Minor: Perhaps you can adjust the name of the script file to something like: `check-test-service-for-changes.ts` then it is clear from the name what the script does.

**Review by FrankEssenberger — APPROVED** [review]
LGTM only a very little comment

**Review by marikaner — APPROVED** [review]
I like it! We might want to consider to actually update the test services on this branch based on this instead of having additional manual steps.

**Pull request merged**